### DISCUSSION TITLE
Nuxt 3 preview without build

### DISCUSSION THREAD

**Discussion by johan-Rm** [nuxt/nuxt#16269] [Questions]
Is it possible to preview  nuxt app when data changes without doing build ?

**Comment by ennioVisco** [comment]
Just run `yarn dev`

**Comment by johan-Rm** [comment]
oh yes I forgot to specify, thank you for your answer, I meant in PROD mode with a vhost like something like that
https://domain.com/?preview=true

**Reply by ennioVisco** [reply]
Did you try with `nuxi preview`? What issues do you get? https://nuxt.com/docs/api/commands/preview#nuxi-preview

**Reply by TheAlexLichter** [reply]
@johan-Rm That feature (as it was in Nuxt 2) does not exist right now. Feel free to create a feature request!
